lib/api/controllers/funnelController.js

```diff
@@ -279,46 +280,48 @@ class FunnelController {
    * @param {KuzzleError|*} err
    */
   handleErrorDump(err) {
-    if (this.kuzzle.config.dump.enabled && this.kuzzle.config.dump.handledErrors.enabled) {
-      setImmediate(() => {
-        const
-          lastSeen = Date.now(),
-          errorType = typeof err === 'object' && err.name ? err.name : typeof err;
-
-        if (this.kuzzle.config.dump.handledErrors.whitelist.indexOf(errorType) > -1) {
-          const request = new Request({
-            controller: 'actions',
-            action: 'dump',
-            body: {}
-          });
+    const handledErrors = this.kuzzle.config.dump.handledErrors;
 
-          // JSON.stringify(new NativeError()) === '{}', i.e. Error, SyntaxError, TypeError, ReferenceError, etc.
-          this.kuzzle.log.error(err instanceof Error && !(err instanceof KuzzleError)
-            ? err.message + '\n' + err.stack
-            : err
-          );
+    if (this.kuzzle.config.dump.enabled && handledErrors.enabled) {
+      setImmediate(() => {
+        const errorType = typeof err === 'object' && err.name ?
+          err.name :
+          typeof err;
+
+        if (handledErrors.whitelist.indexOf(errorType) > -1) {
+          const now = Date.now();
+
+          // JSON.stringify(new NativeError()) === '{}'
+          // i.e. Error, SyntaxError, TypeError, ReferenceError, etc.
+          this.kuzzle.log.error(
+            err instanceof Error && !(err instanceof KuzzleError)
+              ? err.message + '\n' + err.stack
+              : err);
+
+          if (
+            !this.lastDumpedErrors[errorType] ||
+            this.lastDumpedErrors[errorType] < now - handledErrors.minInterval
+          ) {
+            // simplify error message to use it in folder dump name
+            let errorMessage = err.message;
```

Comment:
Much simpler somethink like this, no?
```javascript
const errorMessage = (errorMessage.indexOf('\n') > -1 ? errorMessage.split('\n')[0] : err.message)
  .toLowerCase()
  .replace(/[^a-zA-Z0-9-_]/g, '-')
  .replace(/[-]+/g, '-')
  .split('-')
  .filter(value => value !== '')
  .join('-');
```

Reply:
I think the last `.replace()` is useless because the last operations (split, filter, join) do the same thing

Reply:
Ok for the last replace, but I think that using ternaries in complex, multilines statements is much harder to read.